src/main/java/seedu/address/model/task/Task.java

```diff
@@ -116,7 +108,7 @@ public String toString() {
                 .append("Description: ")
                 .append(getDescription() + "\n")
                 .append(hasDate() ? " Time: " : "")
-                .append(hasTime() ? getDateTime() : getDate());
+                .append(getDateTime());
```

Comment:
I think hasTime() ? getDateTime().get() : "" is better as it is better to show String rather than Optional[something].

Reply:
I will change it to something like this
 ```
   public String toString() {
        final StringBuilder builder = new StringBuilder();
        builder.append("Title: ")
                .append(getTitle() + "\n")
                .append(getDescriptionString())
                .append(getDateTimeString());
        return builder.toString();
    }
```
where getDescriptionString() is:
```
    private StringBuilder getDescriptionString() {
        StringBuilder emptyString = new StringBuilder("");
        return description.map(desc ->
                new StringBuilder("Description: ")
                        .append(desc.toString() + "\n")).orElse(emptyString);
    }
```
and getDateTimeString() is:
```
    private StringBuilder getDateTimeString() {
        StringBuilder emptyString = new StringBuilder("");
        return dateTime.map(time ->
                new StringBuilder("Time: ")
                        .append(time.toString() + "\n")).orElse(emptyString);
    }
```

Reply:
Great.